// protractor/example/conf.js
// An example configuration file.
var jasmineReporters = require('jasmine-reporters');
exports.config = {

  directConnect: true,

  // Capabilities to be passed to the webdriver instance.
  capabilities: {
    'browserName': 'chrome'
  },

  //The address of a running Selenium server
  seleniumAddress: 'http://localhost:4444/wd/hub',

  // Framework to use. Jasmine 2 is recommended.
  framework: 'jasmine2',

  // Spec patterns are relative to the current working directly when
  // protractor is called.
  specs: ['./tests/CreateGroupTest.js'],

  //this can be replaced by localhost address to run tests locally, the following address is the old staging server address
  baseUrl: 'http://131.234.31.122:9000',

  // Options to be passed to Jasmine-node.
  onPrepare: function () {
    jasmine.getEnv().addReporter(new jasmineReporters.JUnitXmlReporter({
      savePath: './outputs/',
      consolidateAll: false
    }));
  },
  jasmineNodeOpts: {
    showColors: true,
    defaultTimeoutInterval: 600000000,
    print: function() {}
  }
};
